### PR TITLE
jwto: init at 0.3.0

### PR DESCRIPTION
**Pull request by jtcoolen**
#### Motivation for this change
[jwto](https://github.com/sporto/jwto): JSON Web Tokens (JWT) encoding, decoding and verification

###### Things done

- [] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/ocaml-modules/jwto/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  propagatedBuildInputs =
+    [ alcotest cryptokit fmt yojson base64 re ppx_deriving ];
+
+  doCheck = true;
+
+  meta = {
+    homepage = "https://github.com/sporto/jwto";
+    description = "JSON Web Tokens (JWT) for OCaml.";
```
Comment (IvarWithoutBones) [inline pkgs/development/ocaml-modules/jwto/default.nix:24]: `meta.description` should not end with a period, as per the [contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
```suggestion
    description = "JSON Web Tokens (JWT) for OCaml";
```

File: pkgs/development/ocaml-modules/jwto/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+buildDunePackage rec {
+  pname = "jwto";
+  version = "0.3.0";
+
+  minimumOCamlVersion = "4.05";
+
+  src = fetchFromGitHub {
+    owner = "sporto";
+    repo = "jwto";
+    rev = "${version}";
```
Comment (IvarWithoutBones) [inline pkgs/development/ocaml-modules/jwto/default.nix:13]:
```suggestion
    rev = version;
```

**Review by IvarWithoutBones — COMMENTED** [review]
A few nits, otherwise LGTM :+1: 

result of `nixpkgs-review pr 95173`:
```
1 package built:
ocamlPackages.jwto
```

File: pkgs/development/ocaml-modules/jwto/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    sha256 = "1p799zk8j9c0002xzi2x7ndj1bzqf14744ampcqndrjnsi7mq71s";
+  };
+
+  propagatedBuildInputs =
+    [ alcotest cryptokit fmt yojson base64 re ppx_deriving ];
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/jwto/default.nix:18]: The library `alcotest` belongs to `checkInputs` rather than `propagatedBuildInputs`.

**Review by vbgl — APPROVED** [review]

**Pull request closed without merging**